src/main/resources/application.yaml

```diff
@@ -6,42 +6,27 @@ logging:
     io.saagie: WARN
     me.ramswaroop.jbot: WARN
 ## GENERAL
-url: ${ASTON_PARKING_URL}
+url: ${ASTON_PARKING_URL:test}
 # SLACK BOT
-slackBotToken: ${ASTON_PARKING_SLACKBOT_TOKEN}
-slackVerificationToken: ${ASTON_PARKING_SLACKBOT_VERIFICATION_TOKEN}
-slackWebhookUrl: ${ASTON_PARKING_SLACK_WEBHOOK_URL}
+slackBotToken: ${ASTON_PARKING_SLACKBOT_TOKEN:test}
+slackVerificationToken: ${ASTON_PARKING_SLACKBOT_VERIFICATION_TOKEN:test}
+slackWebhookUrl: ${ASTON_PARKING_SLACK_WEBHOOK_URL:test}
 rtmUrl: "https://slack.com/api/rtm.start?token={token}&simple_latest&no_unreads"
-# MAIL
-sendEmail: true
-mailFrom: ${ASTON_PARKING_MAIL_FROM}
-spring:
-  mail:
-    host: ${ASTON_PARKING_MAIL_HOST}
-    port: ${ASTON_PARKING_MAIL_PORT}
-    username: ${ASTON_PARKING_MAIL_USERNAME}
-    password: ${ASTON_PARKING_MAIL_PASSWORD}
-    protocol: smtp
-    properties:
-          mail:
-            smtp:
-              starttls:
-                enable: false
-              auth: false
-              connectiontimeout: 5000
-              timeout: 5000
-              writetimeout: 5000
+user:
+  username: ${ASTON_PARKING_ACTIONS_USERNAME:googleActionsUser}
+  password: ${ASTON_PARKING_ACTIONS_PASSWORD:vv9OLtNg8K9hZGZmA60sv47F}
 
+spring:
 # MONGO
   data:
     mongodb:
-      uri: mongodb://${ASTON_PARKING_MONGO_URI}
+      uri: mongodb://${ASTON_PARKING_MONGO_URI:astonparking:O0x2oYi7xLMX0GFk1AzqrZ1uz2FaUL78N0sfgAsnQu@192.168.57.30:27017/astonparking}
 ##########################
 # SECURITY
 slack:
   client:
-    clientId: ${ASTON_PARKING_CLIENTID}
-    clientSecret: ${ASTON_PARKING_CLIENTSECRET}
+    clientId: ${ASTON_PARKING_CLIENTID:test}
```

Comment:
Need a default value ?

Reply:
I thought I need it to run ocally but I didn't need so I removed default values for slack